docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./.docker/volumes/postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: feedbackr
+      POSTGRES_PASSWORD: superpass
+  api:
+    build: ./api/
```

Comment:
```suggestion
  api:
    platform: linux/x86_64
    build: ./api/
```

Reply:
remember the nokogiri error found

https://stackoverflow.com/questions/73047662/errors-of-nokogiri-load-error-with-docker-compose-up